cmd/checker/checker.go

```diff
@@ -0,0 +1,59 @@
+package checker
+
+import (
+	"kool-dev/kool/cmd/builder"
+	"log"
+)
+
+// Checker defines the check kool dependencies method
+type Checker interface {
+	CheckKoolDependencies() (string, error)
+}
+
+// DefaultChecker holds commands to be checked.
+type DefaultChecker struct {
+	DockerCmd builder.Executor
+	dockerComposeCmd builder.Executor
+}
+
+// NewChecker initializes checker
+func NewChecker() Checker {
+	var (
+		err error
+		dockerInfoCmd *builder.Command
+		dockerComposePsCmd *builder.Command
+	)
+
+	dockerInfoCmd, err = builder.ParseCommand("docker info")
```

Comment:
Since this is a piece of code we "trust", I think it is best we build the command instead of parsing it out of a string. This is because of load at compile time instead of runtime (parsing the string).

We could have:

```go
dockerInfoCmd = &builder.Command{"docker", []string{"info"}}
```

Reply:
So there is no error to be handled.